Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine
+
+COPY exporter /exporter
```

Comment:
Is this assuming that the exporter is already built?

Should we not add a go build command before it?